Clean up destructor code

rotu opened the pull request:
Much of the destructor code is unnecessary as the default destructor calls member destructors in reverse order. Where necessary, rearranged fields to make sure this order is the correct one.

rotu committed:
Clean up destructor code

rotu committed:
Uncrustify

SteveMacenski reviewed (approved):
I'm fine with this. I'll need to whip out a sensor for a few hours to validate.

SteveMacenski commented:
@rotu I was going to test this today but it looks like you have merge conflicts. Can you resolve those?

rotu committed:
Merge branch 'eloquent-devel' into dtors

rotu committed:
Fix order of fields

Declare active fields toward the end. Fields depending on other fields should appear in dependency order, so that they are destroyed correctly

SteveMacenski committed:
Merge branch 'eloquent-devel' into dtors

SteveMacenski commented:
Fixed another set of merge conflicts from other PR

SteveMacenski commented:
@rotu friendly ping
